plugins/chat/spec/system/page_objects/chat/chat.rb

```diff
@@ -21,6 +21,10 @@ def open_from_header
         find(".chat-header-icon").click
       end
 
+      def has_right_header_href?(href)
```

Comment:
Is "right" the same as "correct" here? Because my initial interpretation was directional, i.e. opposite of "left". Maybe just `#has_header_href?` is sufficient? 🙂